echidna/core/spectra.py

```diff
@@ -0,0 +1,90 @@
+import numpy
+
+class Spectra(object):
+    """ This class contains a spectra as a function of energy, radius and time.
+    
+    The spectra is stored as histogram binned in energy, x, radius, y, and 
+    time, z. This histogram can be flattened to 2d (energy, radius) or 1d 
+    (energy).
+
+    Attributes:
+      _energy_low (float): Lowest bin edge in MeV
+      _energy_high (float): Highest bin edge in MeV
+      _energy_bins (int): Number of energy bins
+      _radial_low (float): Lowest bin edge in mm
+      _radial_high (float): Highest bin edge in mm
+      _radial_bins (int): Number of raidal bins
+      _time_low (float): Lowest bin edge in years
+      _time_high (float): Highest bin edge in years
+      _time_bins (int): Number of time bins
+    """
+    _energy_low = 0.0 # MeV
+    _energy_high = 10.0 # MeV
+    _energy_bins = 1000
+    _radial_low = 0.0 # mm
+    _radial_high = 6000.0 # mm
+    _radial_bins = 600
+    _time_low = 0.0 # years
+    _time_high = 10.0 # years
+    _time_bins = 10
+    def __init__(self, name):
+        """ Initialise the spectra data container.
+
+        Args:
+          name (str): The name of this spectra
+
+        Attributes:
+          _data (:class:`numpy.ndarray`): The histogram of data
+          _name (str): The name of this spectra
+        """
+        self._data = numpy.ndarray(shape=(Spectra._energy_bins, Spectra._radial_bins, Spectra._time_bins), dtype=float)
```

Comment:
Just a suggestion, I think this line would read better if split over multiple lines e.g.

``` python
self._data = numpy.ndarray(shape=(Spectra._energy_bins,
                                  Spectra._radial_bins,
                                  Spectra._time_bins), 
                           dtype=float)
```

Also you could combine this line and the next using the `numpy.zeros` method.